data/data-pipeline/data_pipeline/tests/test_score_output.py

```diff
@@ -0,0 +1,252 @@
+from pathlib import Path
+import pandas as pd
+import pytest
+import data_pipeline.score.field_names as field_names
+from data_pipeline.utils import get_module_logger
+from data_pipeline.config import settings
+from data_pipeline.score.score_narwhal import ScoreNarwhal
+
+logger = get_module_logger(__name__)
+
+
+# this is so slow?
```

Comment:
This specific fixture takes a long time to load... any improvements?

Reply:
Once it's loaded once, will it change? If not, we could `lru_cache` it to at least only ever have to load it once per thread. The other option would be to write a faster to load file format, like feather --- but that has problems of its own, since it could mask problems with the CSV or create new ones.

Reply:
It does only get loaded a single time, so I think `lru_cache` will work. Do you happen to have an example of this handy? (if not, I am happy to google)

Reply:
I have a great example! See https://github.com/usds/justice40-tool/blob/emma-nechamkin/release/score-narwhal/data/data-pipeline/data_pipeline/etl/sources/geo_utils.py#L13 --- for a use like this, you can just import it from functools and then decorate the fixture and it should work. First load will be slow, but subsequent loads should come out of memory, unless pytest fixtures do dark magic.